locust/static/locust.js

```diff
@@ -256,6 +265,8 @@ $('#convert_csv_btn').click(function(){
             success: function(response){
                 if (response.success) {
                     try{
+                        if (response.key_pass_status)
+                            alert(response.missing_key_message);
```

Comment:
is it possible to make flash message instead of alert? flash message is better for this kind of message since alert blocked the UI and once closed we can't read it again